src/pages/tokens/[tokenid].js

```diff
@@ -0,0 +1,330 @@
+              />
+              <VerifiedBadge
+                color="secondary"
+                badgeName={`${token.claim ? 'Claimed' : 'Unclaimed'}`}
+              />
+            </Box>
+          </Box>
+        </Portal>
+      )}
+
+      {!isMobile && (
+        <Box sx={{ mt: 6 }}>
+          <Typography variant="h2" sx={{ fontSize: 20 }}>
+            Token - #{token.id}
+          </Typography>
+          <Box sx={{ mt: 2 }}>
+            <Box
+              sx={{
+                display: 'flex',
+                gap: 2,
+                mt: 2,
+              }}
+            >
+              <VerifiedBadge
+                color="primary"
+                verified={token.claim}
+                badgeName={`${token.claim ? 'Claimed' : 'Unclaimed'}`}
+              />
+              <VerifiedBadge
+                color="secondary"
+                badgeName={`${token.claim ? 'Claimed' : 'Unclaimed'}`}
+              />
+            </Box>
+          </Box>
+        </Box>
+      )}
+
+      <Box
+        sx={[
+          {
+            mt: [6, 14],
+          },
+        ]}
+      >
+        <InformationCard1
+          entityName={token.tree_species_name}
+          entityType="Token Tree"
+          buttonText="View Token Tree"
+          cardImageSrc={token?.tree_image_url}
+          link={`/trees/${token.tree_id}`}
+        />
+      </Box>
+      <Box
+        sx={{
+          mt: [4, 10],
+        }}
+      >
+        <InformationCard1
+          entityName={`${wallet.name} `}
+          entityType="Wallet"
+          buttonText="View the Wallet"
+          cardImageSrc={wallet?.logo_url}
+          link={`/wallets/${wallet.id}`}
+        />
+      </Box>
+      <Typography
+        variant="h4"
+        sx={[
+          {
+            fontSize: [24, 28],
+            lineHeight: (t) => [t.spacing(7.25), t.spacing(8.5)],
+            mt: (t) => [t.spacing(14), t.spacing(18)],
+          },
+        ]}
+      >
+        Token Info
+      </Typography>
+      <Box className={classes.tabBox}>
+        <TreeTag
+          TreeTagValue={new Date(token.created_at).toLocaleDateString()}
+          title="Created At"
+          icon={<img src={calendarIcon} alt="calendar" />}
+        />
+        <TreeTag
+          TreeTagValue={token.id}
+          title="Token ID"
+          icon={<img src={tokenIcon} alt="token" />}
+        />
+      </Box>
+      <Box height={20} />
+    </Box>
+  );
+}
+
+export async function getStaticProps({ params }) {
+  const { tokenid } = params;
+  const token = await getTokenById(tokenid);
+  const { wallet_id } = token;
+  const wallet = await getWalletById(wallet_id);
+
+  return {
+    props: {
+      token,
+      wallet,
+    },
+    revalidate: Number(process.env.NEXT_CACHE_REVALIDATION_OVERRIDE) || 180,
+  };
+}
+
+// eslint-disable-next-line require-await
+export async function getStaticPaths() {
+  return {
+    paths: [],
+    fallback: 'blocking',
+  };
+}
```

Comment:
@Mohmn please change this to the new pattern of getServerSideProp, I temporarily removed the `cache` feature, and plan to use it in the future when we are ready, this feature is potentially causing problems.

Reply:
sorry for the delay
will update it